src/Markup/Perspex.Markup/Data/PropertyAccessorNode.cs

```diff
@@ -50,11 +50,18 @@ protected override void SubscribeAndUpdate(WeakReference reference)
 
             if (instance != null && instance != PerspexProperty.UnsetValue)
             {
-                var plugin = ExpressionObserver.PropertyAccessors.FirstOrDefault(x => x.Match(reference));
+                var accessorPlugin = ExpressionObserver.PropertyAccessors.FirstOrDefault(x => x.Match(reference));
 
-                if (plugin != null)
+                if (accessorPlugin != null)
                 {
-                    _accessor = plugin.Start(reference, PropertyName, SetCurrentValue);
+                    _accessor = accessorPlugin.Start(reference, PropertyName, SetCurrentValue);
+                    foreach (var validationPlugin in ExpressionObserver.ValidationCheckers.Where(x => x.Match(reference)))
```

Comment:
Validation plugins are always added, but their status reports are excluded in ExpressionObserver (line 224 in the changes). The reason we do not always validate is for two reasons:

1. Match behavior of WPF where you can choose your validation schemes.
2. Tracking all bindings would result in all of the messages throwing off the validation tracking in the Control class. (I can make it work with validation turned on for all bindings, but I think it will be less performant and not completely worth it).
3. We could have an all or nothing system for binding validation. That would work for me.